platform-includes/sourcemaps/overview/javascript.cloudflare.mdx

```diff
@@ -19,9 +19,7 @@ If you want to configure source maps to upload manually, follow the guide for yo
 
 ### Guides for Source Maps
 
-- <PlatformLink to="/sourcemaps/uploading/typescript/">
```

Comment:
@AbhiPrasad this was causing a visual glitch on https://docs.sentry.io/platforms/javascript/guides/cloudflare/sourcemaps/

![image](https://github.com/user-attachments/assets/da4c2ca0-8159-4839-85b0-c59a8f136110)